examples/CalendarCardExpansionExample.m

```diff
@@ -40,82 +40,68 @@ @implementation CalendarCardExpansionExampleViewController {
 - (void)didTap {
   _expanded = !_expanded;
 
-  CalendarChipTiming timing = (_expanded
-                               ? CalendarChipMotionSpec.expansion
-                               : CalendarChipMotionSpec.collapse);
+  id<CalendarChipTiming> traits = (_expanded
+                                   ? CalendarChipMotionSpec.expansion
+                                   : CalendarChipMotionSpec.collapse);
 
   MDMMotionAnimator *animator = [[MDMMotionAnimator alloc] init];
   animator.shouldReverseValues = !_expanded;
   animator.beginFromCurrentState = YES;
 
-  [animator animateWithTiming:timing.navigationBarY animations:^{
+  [animator animateWithTraits:traits.navigationBarY animations:^{
     [self.navigationController setNavigationBarHidden:_expanded animated:YES];
   }];
 
   CGRect chipFrame = [self frameForChip];
   CGRect headerFrame = [self frameForHeader];
 
   // Animate the chip itself.
-  [animator animateWithTiming:timing.chipHeight
-                      toLayer:_chipView.layer
-                   withValues:@[ @(chipFrame.size.height), @(headerFrame.size.height) ]
-                      keyPath:MDMKeyPathHeight];
-  [animator animateWithTiming:timing.chipWidth
-                      toLayer:_chipView.layer
-                   withValues:@[ @(chipFrame.size.width), @(headerFrame.size.width) ]
-                      keyPath:MDMKeyPathWidth];
-  [animator animateWithTiming:timing.chipWidth
-                      toLayer:_chipView.layer
-                   withValues:@[ @(CGRectGetMidX(chipFrame)), @(CGRectGetMidX(headerFrame)) ]
-                      keyPath:MDMKeyPathX];
-  [animator animateWithTiming:timing.chipY
-                      toLayer:_chipView.layer
-                   withValues:@[ @(CGRectGetMidY(chipFrame)), @(CGRectGetMidY(headerFrame)) ]
-                      keyPath:MDMKeyPathY];
-  [animator animateWithTiming:timing.chipHeight
-                      toLayer:_chipView.layer
-                   withValues:@[ @([self chipCornerRadius]), @0 ]
-                      keyPath:MDMKeyPathCornerRadius];
+  [animator animateWithTraits:traits.chipHeight between:@[ @(chipFrame.size.height),
+                                                           @(headerFrame.size.height) ]
+                        layer:_chipView.layer keyPath:MDMKeyPathHeight];
```

Comment:
Is it not preferred to keep one argument per line here?

Reply:
Done.